pkg/zng/docs/spec.md

```diff
@@ -153,18 +153,38 @@ are scoped to the stream in which the typedef occurs.
 Type codes for the "scalar types" need not be defined with typedefs and
 are predefined as follows:
 
-| Type       | Code | Type       | Code |
-|------------|------|------------|------|
-| `bool`     |   0  | `bytes`    |  10  |
-| `byte`     |   1  | `bstring`  |  11  |
-| `int16`    |   2  | `enum`     |  12  |
-| `uint16`   |   3  | `ip`       |  13  |
-| `int32`    |   4  | `port`     |  14  |
-| `uint32`   |   5  | `net`      |  15  |
-| `int64`    |   6  | `time`     |  16  |
-| `uint64`   |   7  | `duration` |  17  |
-| `float64`  |   8  | `any`      |  18  |
-| `string`   |   9  |            |      |
+<table>
```

Comment:
I did a double-take when I saw the originally-formatted table, since Markdown made this look like a four-column table rather than two sets of dual-column table formatted to take up less vertical space. I did some research and found [this StackOverflow post](https://stackoverflow.com/questions/43232279/how-can-one-display-tables-side-by-side-in-github-markdown) that shows how a smattering of HTML improves the formatting a bit.